src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -218,6 +220,11 @@
     public static final String MSG_MULTIPLE_LINES_INSIDE =
             "empty.line.separator.multiple.lines.inside";
 
+    /** List of AST token types, which can not have comment nodes to check inside. */
```

Comment:
it is better say that we do not validate inside comment in such tokens, as it does not make sense.
But such tokens can have comments inside by grammar and it is compilable code, it is weird and ugly but compilable.
We are ok to skip them.